### DISCUSSION TITLE
如何回退版本

### DISCUSSION THREAD

**Discussion by RrrrLin** [the1812/Bilibili-Evolved#3655] [提问 / Q&A]
2.4.0更新提示的时候顺手就点了更新，但是现在没法用了，看了下更新日志，那个地址没法访问，怎么退回到之前的2.3.x的旧版本？

**Comment by the1812** [comment]
你之前用的哪个版本

**Reply by RrrrLin** [reply]
具体不太清楚，一般油猴自动检查 提示更新就点了，应该是上一个正式版本？

**Reply by the1812** [reply]
[bilibili-evolved.v2.2.4.zip](https://github.com/the1812/Bilibili-Evolved/files/9482809/bilibili-evolved.v2.2.4.zip)
用这里面的内容覆盖一下油猴里的

然后去脚本里关掉更新相关的功能
![image](https://user-images.githubusercontent.com/26504152/188274620-4a102fa3-39de-49a5-84ad-43edd078f040.png)

**Reply by mirion-dev** [reply]
> 你之前用的哪个版本

jsdelivr不是已经可以重新访问了吗👀

**Reply by RrrrLin** [reply]
> [bilibili-evolved.v2.2.4.zip](https://github.com/the1812/Bilibili-Evolved/files/9482809/bilibili-evolved.v2.2.4.zip) 用这里面的内容覆盖一下油猴里的
> 
> 然后去脚本里关掉更新相关的功能 ![image](https://user-images.githubusercontent.com/26504152/188274620-4a102fa3-39de-49a5-84ad-43edd078f040.png)

非常感谢，提问之后过了一天突然好了，就忘了回复了。
